File: app/src/main/java/com/seankim/zomatosearchex/Model/RspCurrLocationModel.java
package com.seankim.zomatosearchex.Model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RspCurrLocationModel {

    @SerializedName("entity_type")
    @Expose
    String entity_type;

    @SerializedName("entity_id")
    @Expose
    Integer entity_id;

    @SerializedName("title")
    @Expose
    String title;

    @SerializedName("latitude")
    @Expose
    Double latitude;

    @SerializedName("longitude")
    @Expose
    Double longitude;

    @SerializedName("city_id")
    @Expose
    Integer city_id;

    @SerializedName("city_name")
    @Expose
    String city_name;

    @SerializedName("country_id")
    @Expose
    Integer country_id;

    @SerializedName("country_name")
    @Expose
    String country_name;

    public String getEntity_type() {
        return entity_type;
    }

    public void setEntity_type(String entity_type) {
        this.entity_type = entity_type;
    }

    public Integer getEntity_id() {
        return entity_id;
    }

    public void setEntity_id(Integer entity_id) {
        this.entity_id = entity_id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public Double getLatitude() {
        return latitude;
    }

    public void setLatitude(Double latitude) {
        this.latitude = latitude;
    }

    public Double getLongitude() {
        return longitude;
    }

    public void setLongitude(Double longitude) {
        this.longitude = longitude;
    }

    public Integer getCity_id() {
        return city_id;
    }

    public void setCity_id(Integer city_id) {
        this.city_id = city_id;
    }

    public String getCity_name() {
        return city_name;
    }

    public void setCity_name(String city_name) {
        this.city_name = city_name;
    }

    public Integer getCountry_id() {
        return country_id;
    }

    public void setCountry_id(Integer country_id) {
        this.country_id = country_id;
    }

    public String getCountry_name() {
        return country_name;
    }

    public void setCountry_name(String country_name) {
        this.country_name = country_name;
    }
}

File: app/src/main/java/com/seankim/zomatosearchex/Model/RspCurrLocationModel.kt
package com.seankim.zomatosearchex.Model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RspCurrLocationModel {

    @SerializedName("entity_type")
    @Expose
    lateinit var entity_type: String

    @SerializedName("entity_id")
    @Expose
    var entity_id: Int? = null

    @SerializedName("title")
    @Expose
    lateinit var title: String

    @SerializedName("latitude")
    @Expose
    var latitude: Double? = null

    @SerializedName("longitude")
    @Expose
    var longitude: Double? = null

    @SerializedName("city_id")
    @Expose
    var city_id: Int? = null

    @SerializedName("city_name")
    @Expose
    lateinit var city_name: String

    @SerializedName("country_id")
    @Expose
    var country_id: Int? = null

    @SerializedName("country_name")
    @Expose
    lateinit var country_name: String
}
